specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/settings.json

```diff
@@ -86,6 +86,13 @@
           "summary": "Get specified account setting value.",
           "description": "Retrieves the value of a specified, value account setting.",
           "parameters": [
+            {
+              "name": "setting-name",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The name of the account setting. Must be a valid settings option."
+            },
```

Comment:
Can you add this to "parameters" and just ref it in both places, like you ref the ApiVersionParameter from common.json (just put the new one in this doc, though). That's the best way to make sure the definition remains consistent across endpoints.